openfisca_france_inheritance/parameters/droits_mutation_titre_gratuit/pacte_dutreil/duree_conservation_titre_post_transmission.yaml

```
@@ -0,0 +1,39 @@
+description: Pacte Dutreil - Durée de conservation des titres (Post-transmission), condition 2
+values:
+  2003-08-05:
+    value: 6
+  2005-08-03:
+    value: 6
+  2007-09-26:
+    value: 4
+  2007-12-28:
+    value: 4
+  2011-07-30:
+    value: 4
+metadata:
+  short_label: Durée de conservation des titres (Post-transmission)
+  last_value_still_valid_on: "2023-04-01"
```

```suggestion
  last_value_still_valid_on: "2025-06-12"
```

https://avocat-droit-succession-cahen.fr/avant-prevoir/la-transmission-du-patrimoine-professionnel-le-pacte-dutreil/